### ISSUE TITLE
Inverted x/y axis 

### ISSUE THREAD

**Issue by catslab** [danielgindi/Charts#2537]
Hi,
I'm trying to line graph data with X data being a function of Y, instead of the traditional y=f(x).
This is because I need to represent values by depth. More natural to have depth on Y axis.
At each depth on the reversed Y axis ( 0 being on top and maximum depth at bottom), I need to represent on X axis the values of said granularity, temperature and so on.
I tried to swap data but no good. Display is erratic.
Can this be done with this nice library ? 
Thanks.

**Comment by liuxuan30** [comment]
Kind of lost what you asked. Are you asking 
>  /// flag that indicates if the axis is inverted or not
    open var inverted = false
?

**Comment by catslab** [comment]
Ideally I'd need to rotate a standard line graph 90˚ clockwise, thus having Y axis from left to right and X axis from top to bottom.
Am I clearer ? :)

**Comment by liuxuan30** [comment]
not really.. as the x axis can be on top or bottom already and there is already leftAxis and rightAxis.. Check ChartsDemo.

**Comment by thincamel** [comment]
i have the same issue...
for example, i wanted to draw linechart of temperature for underground(like a well), the point order by Y axis.

x = -2.67 depath = -0.50
x = 0.11 depath = -4.10
x = 4.22 depath = -7.70
x = 3.05 depath = -11.30

The connection to these points is wrong

**Comment by thincamel** [comment]
please see the result by changed chart.inverted
https://code.hcharts.cn/highcharts/hhhhy1

**Comment by catslab** [comment]
Exactly what I'd need ! 
I thought about rotating the viewport but I did not had success.

**Comment by liuxuan30** [comment]
Oh I suddenly realized what are you asking.. Problem solved? I cannot open https://code.hcharts.cn/highcharts/hhhhy1 so don't know the details. Mind to share a bit?
